### PR TITLE
kustomize 3.3.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
Test failure on Catalina (which is a blocker, now that Catalina is released):

```
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<126>.
```

**Comment by chenrui333** [comment]
@BrewTestBot test this please

**Comment by chenrui333** [comment]
Is this could be due to the `notorize` process on Catalina OS?

**Comment by chenrui333** [comment]
Now I have Catalina OS, circle back on this.

File: Formula/kustomize.rb [hunk 1]
```diff
@@ -34,8 +34,6 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/kustomize version")
```
Comment (chenrui333) [inline Formula/kustomize.rb]: The version did not get populated correctly as `kustomize` and ` pluginator` are separate modules.

**Comment by chenrui333** [comment]
Interesting, the test step certainly passed on my Catalina run:

```
14:36:25 ==> /usr/local/Cellar/kustomize/3.3.0/bin/kustomize build /private/tmp/kustomize-test-20191026-5391-yal5l6
14:36:25 /usr/local/Cellar/kustomize/3.3.0/bin/kustomize: /usr/local/Cellar/kustomize/3.3.0/bin/kustomize: cannot execute binary file
14:36:25 Error: kustomize: failed
14:36:25 An exception occurred within a child process:
14:36:25   Test::Unit::AssertionFailedError: <0> expected but was
14:36:25 <126>.
```

**Comment by chenrui333** [comment]
```
==> /usr/local/Cellar/kustomize/3.3.0/bin/kustomize build /private/tmp/kustomize-test-20191030-20931-7zdvy5

$ /usr/local/Cellar/kustomize/3.3.0/bin/kustomize version
Version: {Version:unknown GitCommit:$Format:%H$ BuildDate:1970-01-01T00:00:00Z GoOs:darwin GoArch:amd64}
```

**Comment by fxcoudert** [comment]
I think this is — again — the `tar` issue that is corrupting the Catalina bottle

**Comment by chenrui333** [comment]
Close in favor #47311

**Pull request closed without merging**